Documentation/install/guides/kops.rst

```diff
@@ -0,0 +1,31 @@
+**********************************
+Kubernetes Kops Installation Guide
+**********************************
+
+As of kops 1.9 release, Cilium is supported as CNI plugin for kops-deployed clusters.
+
+Cilium needs newer kernel version than default kops images, so you need to supply an ami which will be new enough for Cilium to run on it.
```

Comment:
... needs a newer kernel version then the default kops images...

Which kernel version? Please specify an exact version minimum.

Reply:
Done